uint32 src_vehicle